contentcuration/contentcuration/static/js/edit_channel/share/views.js

```diff
@@ -53,7 +53,8 @@ var ShareView = BaseViews.BaseView.extend({
     },
     get_share_modes: function(){
         if (!this.share_modes){
-            if(_.find(window.current_channel.get("editors"), function(u){return u === this.current_user.id})){
+            var user_is_editor = _.find(window.current_channel.get("editors"), function(u){return u === this.current_user.id});
```

Comment:
I was gonna say something like:
```
var user_is_editor = _.find( window.current_channel.get("editors"), 
    function (u)  {
        return u === this.current_user.id
    }
);
```

But it doesn't look as good as I thought it would. At this point, it's just preference. Thanks!